.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright The ORAS Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Bug Report
+description: File a bug report
+labels: [bug]
+body:
+  - type: markdown
+    id: preface
+    attributes:
+      value: |
+        Thank you for reporting bugs to ORAS!
+  - type: markdown
+    id: reproduce
+    attributes:
+      value: "## Describe the bug"
+  - type: textarea
+    id: environment
```

Comment:
What does `id` represent? Looks like this id is duplicated in the `Describe your environment` section

Reply:
Good catch. I removed the duplicated id.